custom_components/tuya_local/devices/monzana_klimaanlage_air_conditioner.yaml

```diff
@@ -0,0 +1,78 @@
+name: Air conditioner
+products:
+  - id: 000004y1yr
+    name: Monzana Klimaanlage
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cool
+              value: cool
+            - dps_val: Dyr
```

Comment:
This misspelling is from the original tuya protocol, not a typo?

Reply:
Yes, this misspelling is indeed intentional.